staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_edit_last_applied.go

```diff
@@ -62,8 +63,8 @@ func NewCmdApplyEditLastApplied(f cmdutil.Factory, ioStreams genericclioptions.I
 	cmd := &cobra.Command{
 		Use:                   "edit-last-applied (RESOURCE/NAME | -f FILENAME)",
 		DisableFlagsInUseLine: true,
-		Short:                 "Edit latest last-applied-configuration annotations of a resource/object",
-		Long:                  applyEditLastAppliedLong,
+		Short:                 i18n.T("Edit latest last-applied-configuration annotations of a resource/object"),
+		Long:                  i18n.T(applyEditLastAppliedLong),
```

Comment:
`i18n.T` should be invoked on a text, not on a variable, like everywhere else

Reply:
@soltysh If there is a long text that have variables concatinated inside it, should I wrap the entire text with the translation or only wrap the string literal parts?

Reply:
From what I see there are no variables being injected, and if they where I'm pretty sure that they should not be translated.

Reply:
@soltysh So do I understand you correctly when I say that `i18n.T()` should only ever be applied to string literals, and nothing else?

Reply:
@soltysh Yes I understand that but in [this example](https://github.com/kubernetes/kubernetes/blob/56833a63c902e8d350d7335ef91746f554def967/staging/src/k8s.io/kubectl/pkg/cmd/config/config.go#L43-L50) there are variables intertwined with the text, should the `i18n.T()` wrap the entire thing or only the string literal parts?